apisix/plugins/body-transformer.lua

```diff
@@ -184,6 +201,8 @@ local function set_input_format(conf, typ, ct, method)
             conf[typ].input_format = "json"
         elseif str_find(ct:lower(), "application/x-www-form-urlencoded", nil, true) then
             conf[typ].input_format = "encoded"
+        elseif str_find(ct:lower(), "multipart/", nil) then
```

Comment:
performance: we should enable `plain mode`

<img width="1127" alt="image" src="https://github.com/user-attachments/assets/e66a4b45-b358-4cb1-abbc-3276ce11eead">